src/platform/term.c

```diff
@@ -887,6 +890,32 @@ int term_keycodeByName(const char *name) {
     return name[0];
 }
 
+static int term_ctrlPressed(int* key) {
+    if (*key == 13) {
+        *key = RETURN_KEY;
+        return 0;
+    }
```

Comment:
what is this for?

Reply:
The curses initialization changes I needed to make for this PR changed the keycode returned by the enter key. Added a comment mentioning this.